src/main/java/uk/gov/hmcts/reform/sendletter/services/LetterService.java

```diff
@@ -35,71 +35,91 @@
 
     private static final Logger log = LoggerFactory.getLogger(LetterService.class);
 
-    private final PdfCreator pdfCreator = new PdfCreator(new DuplexPreparator(), new HTMLToPDFConverter()::convert);
+    private final PdfCreator pdfCreator;
     private final LetterRepository letterRepository;
     private final Zipper zipper;
     private final ObjectMapper mapper;
 
-    public LetterService(LetterRepository letterRepository, Zipper zipper, ObjectMapper mapper) {
+    public LetterService(
+        PdfCreator pdfCreator,
+        LetterRepository letterRepository,
+        Zipper zipper,
+        ObjectMapper mapper
+    ) {
+        this.pdfCreator = pdfCreator;
         this.letterRepository = letterRepository;
         this.zipper = zipper;
         this.mapper = mapper;
     }
 
-    @Transactional
-    public UUID send(LetterRequest letter, String serviceName) {
-        return send(
-            new LetterInfo(generateChecksum(letter), serviceName, letter.type, letter.additionalData),
-            () -> pdfCreator.createFromTemplates(letter.documents)
+    // TODO: remove
+    @Autowired
+    public LetterService(
+        LetterRepository letterRepository,
+        Zipper zipper,
+        ObjectMapper mapper
+    ) {
+        this(
+            new PdfCreator(new DuplexPreparator(), new HTMLToPDFConverter()::convert),
+            letterRepository,
+            zipper,
+            mapper
```

Comment:
leaving this one so that e2e test, and other, pass (PdfCreator can't be autowired as it's in a separate package)